src/xdocs/config_javadoc.xml

```diff
@@ -410,13 +410,13 @@ public int field;
         </p>
 
         <p>
-          Error messages about parameters and type parameters
+          Violate parameters and type parameters
```

Comment:
`Violate` => `Violations about`.

There are 2 more in this same file afterwards.

Reply:
please expand context of diff to see previous paragraph.
It Checks ..
It Violate ...

https://www.merriam-webster.com/dictionary/violate

Reply:
One more change:
```
         <p>
-          Violate parameters and type parameters
+          Violates parameters and type parameters
           for which no param tags are
           present can be suppressed by defining property
-          <code>allowMissingParamTags</code>. Violate
+          <code>allowMissingParamTags</code>. Violates
           exceptions which are declared to be thrown, but for which no
           throws tag is present can be suppressed by defining property
-          <code>allowMissingThrowsTags</code>. Violate
+          <code>allowMissingThrowsTags</code>. Violates
           methods which return non-void but for which no return tag is
           present can be suppressed by defining property
```

Reply:
I am fine with the new changes. `Violate` sounded funny to me, since it was used like a verb and it didn't sound like a verb should be there.